src/course-page-resources/course-page/CoursePageConfigCard.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCog } from '@fortawesome/free-solid-svg-icons';
+import { Button } from '@edx/paragon';
+import classNames from 'classnames';
+
+import messages from '../messages';
+
+function CoursePageConfigCard({
+  intl,
+  id,
+  title,
+  description,
+  pageStatus,
+  showSettings,
+}) {
+  const cardSettings = 'bg-white shadow p-3 m-0 my-3 course-page-title';
+  const cardClasses = classNames('card', cardSettings, { disabled: !pageStatus });
+  const cardSubSettings = 'bg-white px-3 py-0';
+
+  const buttonSettings = 'd-flex mt-0 justify-content-center align-items-center';
+  const buttonClasses = classNames(buttonSettings, { visible: !pageStatus, invisible: pageStatus });
+  const pageStatusMsgId = pageStatus ? 'pageStatus.enabled' : 'pageStatus.disabled';
+
+  return (
+    <div className={cardClasses} key={id}>
+      <div className={classNames('card-header', cardSubSettings)}>
+        <div className="row">
+          <h2 className="font-weight-bold">{title}</h2>
+          <div className="col flex-grow" />
+          {showSettings && <FontAwesomeIcon icon={faCog} />}
+        </div>
+      </div>
+      <div className={classNames('card-body', cardSubSettings)}>
+        {showSettings && <p>{intl.formatMessage(messages[pageStatusMsgId])}</p>}
```

Comment:
```suggestion
        <span>{intl.formatMessage(messages[pageStatusMsgId])}</span>
```
These are not related, and this is not a para of text.

Reply:
They seem to be related: the page enabled/disabled status is shown only when the settings gear is shown. However, span seems to work nicely after some tweaking of the surroundings.

Reply:
@taniwha The mockup doesn't describe the functionality, it's just a mockup with a few sample cases.